common/persistence/sql/sqlplugin/interfaces.go

```diff
@@ -220,7 +220,7 @@ type (
 	// TasksFilter contains the column names within tasks table that
 	// can be used to filter results through a WHERE clause
 	TasksFilter struct {
-		ShardID              int
+		ShardID              int // this is DBShardID, not historyShardID (TODO: maybe rename it for clarification)
```

Comment:
It's history ShardID.

Reply:
https://github.com/uber/cadence/blob/master/common/persistence/sql/sqlTaskStore.go#L550

Can you confirm? I think this is different from execution store